lib/vrl/stdlib/src/split.rs

```diff
@@ -168,7 +172,43 @@ mod test {
              ],
              want: Ok(value!(["˙ƃuᴉɹʇs", "ƃuol", "ɐ", "sᴉ", "sᴉɥ┴"])),
              tdef: TypeDef::array(Collection::from_unknown(Kind::bytes())),
-         }
+        }
```

Comment:
Small nit, as this is in a macro `rustfmt` fails to format. For consistency could we add a line break between the test case here and then match the indention across everything?
```suggestion
        
       }
```

Reply:
ah the macro makes it fail to format, makes sense

I actually tried to fix [the indentation for this on master](https://github.com/vectordotdev/vector/blob/master/lib/vrl/stdlib/src/split.rs#L171) which has an extra space for the closing bracket, I might be misunderstanding the suggestion

Reply:
tl;dr - I wouldn't worry much about this as long as it's inline with the existing code.